controller/templates/pvc.yaml

```diff
@@ -7,7 +7,11 @@ spec:
     - ReadWriteOnce
   resources:
     requests:
+      {% if storage["size"] is not none and storage["size"] != "" %}
```

Comment:
oh, I didn't know that Jinja uses `is not none` instead of python style `is not None`! 👍